source/CardGames.py

```diff
@@ -302,6 +302,16 @@ def ask_card(self):
     self.player_asked = self.playerinfo[self.index(player_asked, self.playerinfo)][0]
 
     self.surrender_card()
+  def count_cards(self, deck):
```

Comment:
Code looks great Zach. Nailed the task. Thanks for adding comments. I know that's something we all said we would improve for this sprint.

Reply:
Thanks, Andre, and thanks for noticing that.